src/System.Windows.Forms.Primitives/src/Windows/Win32/PInvoke.AreDpiAwarenessContextsEqual.cs

```diff
@@ -0,0 +1,33 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+using System.Windows.Forms;
+
+namespace Windows.Win32
+{
+    internal static partial class PInvoke
+    {
+        /// <summary>
+        ///   Compares two DPI awareness context values for equality.
+        /// </summary>
+        /// <returns>
+        ///   <see langword="true"/> if <paramref name="dpiContextA"/> and <paramref name="dpiContextB"/> are equal;
+        ///   otherwise, <see langword="false"/> if not equal or the underlying OS does not support this API.
+        /// </returns>
```

Comment:
Apologies for the confusion, I guess I didn't make it clear enough.

By default VS adds the same indentation for the text as for the tags
![image](https://user-images.githubusercontent.com/4403806/189260088-0ed5ebd1-f91b-4b85-a3ce-0ef118daf760.png)

...and we add an extra space to help with readability:
```suggestion
        /// <summary>
        ///  Compares two DPI awareness context values for equality.
        /// </summary>
        /// <returns>
        ///  <see langword="true"/> if <paramref name="dpiContextA"/> and <paramref name="dpiContextB"/> are equal;
        ///  otherwise, <see langword="false"/> if not equal or the underlying OS does not support this API.
        /// </returns>
```